corehq/apps/userreports/tasks.py

```diff
@@ -54,7 +54,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
         redis_client.delete(redis_key)
 
 
-@task(queue='ucr_queue', ignore_result=True, acks_late=True)
+@task(queue='ucr_queue', ignore_result=True)
 def rebuild_indicators(indicator_config_id):
```

Comment:
I actually thought we should remove it from this task but not from `resume_building_indicators`. That one doesn't rebuild the table does it?